.github/workflows/lint.yml

```
@@ -9,12 +9,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.5, 3.6, 3.7, 3.8]
+        python-version: ["3.6", "3.7", "3.8", "3.10"]
```

Lint on all python version that PyWebIO support:

```suggestion
        python-version: ["3.5", "3.6", "3.7", "3.8", "3.9", "3.10"]
```

---

Regardless of py3.5, py3.9 should also be added to the lint list, right?

```suggestion
        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
```

---

Let's try to find a reasonable compromise that does not contribute to global warming ;-)  What about 3.5, 3.7, 3.9, and 3.10, or else 3.6, 3.8, and 3.10?  I can not think of any lint errors that would not be caught by that testing.

Python 3.5 is [end-of-life](https://devguide.python.org/devcycle/#end-of-life-branches) and is no longer supported.  The problem in this codebase is that `gomoku_game.py` uses f-strings which are [faster](https://www.scivision.dev/python-f-string-speed/) but flake8 rightly complains that they are Syntax Errors on Python 3.5.